server/bin/pulp-qpid-ssl-cfg

```diff
@@ -14,7 +14,11 @@
 #
 
 DIR="/tmp/tmp$RANDOM"
-HOST=`hostname`
```

Comment:
Since you have introduced logic here, I'd like to see this moved under the ``setup`` section with the others.  This part of the script is for static variables.